content/facets.md

```diff
@@ -0,0 +1,138 @@
+# 12 Factor Facets
+
+While the 12 factors serve as helpful specific guidelines for development teams, the rationale behind the factors is to enable a set of application-platform interactions which simplify the operation of applications which follow the recommendations in the factors.  These interactions can be broadly grouped by the platform behaviors they enable, which serve as “facets” of operation where the 12 factors simplify maintenance.  The facets enable applications which are:
+
+* **Continuously Deployable**: it is easy to deploy new software versions in existing environments
+* **Composable**: it is easy to set up existing software versions in new environments
+* **Scalable**: it is easy to manage the software within an environment
+* **Observable**: it is easy to debug the software when running in an production environment
```

Comment:
Are we missing a Resilient/Fault Tolerant? Emphasizing 6,8,9?

Reply:
I see resilience / reliability as a point along the arc from "scales well" to "can see how it's broken", but I'd generally fold fault tolerance into scalable, with some important asterisks on the application code side.

Reply:
Something like this could be a follow on, so no need to address in this review